README.rdoc

```diff
@@ -53,7 +53,14 @@ Please, have a look at the wiki pages about development workflow and code standa
 
     rake
 
-  *NOTE:* For this you will need the Kalibro Gatekeeper service up and running as explained below.
+  *NOTE:* For this you will need the Kalibro Processor service up and running as explained below.
```

Comment:
I don1t think we depend on Processor by any means. We don't even include client on the Gemfile.